BackgroundProcessing/TranslateWorker/Controllers/TranslateController.cs

```diff
@@ -25,16 +26,16 @@ namespace TranslateWorker.Controllers
     /// <summary>
     /// The message Pubsub posts to our controller.
     /// </summary>
-    public class PostMessage
+    public class PostMessage
     {
         public PubsubMessage message { get; set; }
-        public string subscription { get; set; }
+        public string subscription { get; set; }

```

Comment:
nit: properties should start with capital letters

Reply:
In this case, the message being posted to the uri is a json with lowercase prop names, so I thought it better to match the json.